content/app/how-tos/ingress-with-whitelist.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "Create Ingress with IP Whitelisting"
+weight = 5
+chapter = false
+pre = ""
++++
+
+{{% notice warning %}}
+This uses beta features in the platform and breaking changes may occur in the future
+{{% /notice %}}
+
+## Create middleware
+{{% notice note %}}
+If on your resource you come across a `middleware.traefik.containo.us`, that is an older version of Treaefik's CRD. It will be deprecated in the future.
+{{% /notice %}}
+
+Create a `middleware.traefik.io` object to whitelist IP addresses:
+
+```yaml
+apiVersion: traefik.io/v1alpha1
+kind: Middleware
+metadata:
+  name: <middlewareName>
+  namespace: <middlewareNamespace>
+spec:
+  ipWhiteList:
+    ipStrategy:
+      depth: 2 # depth is required as request is forwarded from load balancer
```

Comment:
not sure what it is, but is depth 2 always the case?

Reply:
ok, that depends on the google impl:https://cloud.google.com/load-balancing/docs/https#x-forwarded-for_header, not sure if we want to add link to that so they know that client_id is at pos 2? if not ignore